internal/serviceapi/clusterapi/resource_schema.go

```diff
@@ -448,7 +529,6 @@ func ResourceSchema(ctx context.Context) schema.Schema {
 											},
 											"ebs_volume_type": schema.StringAttribute{
 												Computed:            true,
-												Optional:            true,
```

Comment:
This change in computability in `analytics_specs.ebs_volume_type` does not look correct, checked in API spec and it is present in the request schema. Will create a follow up ticket so we can look into more detail, we do already have some existing inconsistencies in the autogenerated cluster schema.